app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,37 @@
+class BookingsController < ApplicationController
+  before_action :authenticate_user! unless Rails.env == 'test'
+  load_and_authorize_resource
+
+  def index
+    bookings = Booking.includes(:package).all
+    render json: bookings.as_json(include: :package) unless bookings.empty?
+
+    render json: { message: 'There are no booking created currently.' } if bookings.empty?
+  end
+
+  def new; end
+
+  def create
+    @booking = Booking.new(booking_params)
+    if @booking.save!
```

Comment:
Remember `save!` raises its own exception. Better to use `save` since you are already doing your own error handling